decode/performance.ts

```diff
@@ -108,13 +108,14 @@ export function decode(tokens: Token[]): PerformanceEvent  {
 }
 
 function process(network: any[] | number[], stringIndex: number): NetworkData {
+    let empty = "";
     return {
-        start: network[0] as number,
-        duration: network[1] as number,
-        size: network[2] as number,
+        start: network[0] === -1 ? null : network[0] as number,
```

Comment:
When would this be null?

Reply:
Previous issues that you reported were due to the fact that different browsers implement spec differently. For now, the known field is "transferSize". For instance, not all versions of Edge support it. I've added the check to see if we get null across any browser and if we don't, we can remove the future check. But today if we get null, the decoding will go wrong and it will yield random output like you shared in the email. 

---
In reply to: [368662863](https://github.com/microsoft/clarity-js/pull/166#discussion_r368662863) [](ancestors = 368662863)

Reply:
I've also added comment.

---
In reply to: [368802956](https://github.com/microsoft/clarity-js/pull/166#discussion_r368802956) [](ancestors = 368802956,368662863)

Reply:
Why does decoding go wrong with null values? Null seems like a perfectly legitimate values that the encoding/decoding should handle. 

---
In reply to: [368802993](https://github.com/microsoft/clarity-js/pull/166#discussion_r368802993) [](ancestors = 368802993,368802956,368662863)

Reply:
Simplified this whole code. Not using null anymore. 

---
In reply to: [369054487](https://github.com/microsoft/clarity-js/pull/166#discussion_r369054487) [](ancestors = 369054487,368802993,368802956,368662863)